weblate/addons/views.py

```diff
@@ -206,3 +206,22 @@ def post(self, request, *args, **kwargs):
                 return redirect(reverse("manage-addons"))
             return redirect(reverse("addons", kwargs={"path": target.get_url_path()}))
         return super().post(request, *args, **kwargs)
+
+
+class AddonLogs(DetailView):
```

Comment:
This view is missing permission checks, see `get_object` in `AddonDetail` class. Probably there should be be something `BaseAddonView` where this method would be moved and both `AddonDetail` and `AddonLogs` would inherit from it.